core/src/main/java/org/elasticsearch/script/ScriptModule.java

```diff
@@ -34,57 +35,60 @@
 import java.util.stream.Collectors;
 
 /**
- * An {@link org.elasticsearch.common.inject.Module} which manages {@link ScriptEngineService}s, as well
- * as named script
+ * Manages building {@link ScriptService} and {@link ScriptSettings} from a list of plugins.
  */
-public class ScriptModule extends AbstractModule {
+public class ScriptModule {
```

Comment:
I'm not 100% sure keeping ScriptModule was the right way to go but it *seems* ok. It is no longer an extension point or a Guice module, but something has to build ScriptService and ScriptSettings and this seems a fairly good spot.